packages/contracts-bedrock/test/DeployImplementations.t.sol

```diff
@@ -246,60 +244,66 @@ contract DeployImplementations_Test is Test {
-
-        // Assert that the full input and output structs were properly set.
-        assertEq(keccak256(abi.encode(_input)), keccak256(abi.encode(DeployImplementationsInput(dii).input())), "1500");
-        assertEq(
-            keccak256(abi.encode(output)), keccak256(abi.encode(DeployImplementationsOutput(dio).output())), "1600"
-        );
-
-        // Assert inputs were properly passed through to the contract initializers.
-        assertEq(output.delayedWETHImpl.delay(), _input.withdrawalDelaySeconds, "1700");
-        assertEq(output.preimageOracleSingleton.challengePeriod(), _input.challengePeriodSeconds, "1800");
-        assertEq(output.preimageOracleSingleton.minProposalSize(), _input.minProposalSizeBytes, "1900");
-        assertEq(output.optimismPortalImpl.proofMaturityDelaySeconds(), _input.proofMaturityDelaySeconds, "2000");
-        assertEq(
-            output.optimismPortalImpl.disputeGameFinalityDelaySeconds(), _input.disputeGameFinalityDelaySeconds, "2100"
-        );
+    function hash(bytes32 _seed, uint256 _i) internal pure returns (bytes32) {
+        return keccak256(abi.encode(_seed, _i));
+    }
+
+    function testFuzz_run_memory_succeeds(bytes32 _seed) public {
+        withdrawalDelaySeconds = uint256(hash(_seed, 0));
+        minProposalSizeBytes = uint256(hash(_seed, 1));
+        challengePeriodSeconds = bound(uint256(hash(_seed, 2)), 0, type(uint64).max);
+        proofMaturityDelaySeconds = uint256(hash(_seed, 3));
+        disputeGameFinalityDelaySeconds = uint256(hash(_seed, 4));
+        release = string(bytes.concat(hash(_seed, 5)));
+        superchainConfigProxy = SuperchainConfig(address(uint160(uint256(hash(_seed, 6)))));
+        protocolVersionsProxy = ProtocolVersions(address(uint160(uint256(hash(_seed, 7)))));
+
+        dii.set(dii.withdrawalDelaySeconds.selector, withdrawalDelaySeconds);
+        dii.set(dii.minProposalSizeBytes.selector, minProposalSizeBytes);
+        dii.set(dii.challengePeriodSeconds.selector, challengePeriodSeconds);
+        dii.set(dii.proofMaturityDelaySeconds.selector, proofMaturityDelaySeconds);
+        dii.set(dii.disputeGameFinalityDelaySeconds.selector, disputeGameFinalityDelaySeconds);
+        dii.set(dii.release.selector, release);
+        dii.set(dii.superchainConfigProxy.selector, address(superchainConfigProxy));
+        dii.set(dii.protocolVersionsProxy.selector, address(protocolVersionsProxy));
+
+        deployImplementations.run(dii, dio);
+
+        // Assert that individual input fields were properly set based on the inputs.
+        assertEq(withdrawalDelaySeconds, dii.withdrawalDelaySeconds(), "100");
+        assertEq(minProposalSizeBytes, dii.minProposalSizeBytes(), "200");
+        assertEq(challengePeriodSeconds, dii.challengePeriodSeconds(), "300");
+        assertEq(proofMaturityDelaySeconds, dii.proofMaturityDelaySeconds(), "400");
+        assertEq(disputeGameFinalityDelaySeconds, dii.disputeGameFinalityDelaySeconds(), "500");
```

Comment:
I think there are some assertions missing here.
```suggestion
        assertEq(disputeGameFinalityDelaySeconds, dii.disputeGameFinalityDelaySeconds(), "500");
        assertEq(release, dii.release(), "525");
        assertEq(address(superchainConfigProxy), address(dii.superchainConfigProxy()), "550");
        assertEq(address(protocolVersionsProxy), address(dii.protocolVersionsProxy()), "575");
```

Reply:
Nice finds on the missing assertions, fixed in e949871aa